eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -80,4 +83,22 @@ void existsInDirTest(@TempDir final Path temp) throws IOException {
             Matchers.is(true)
         );
     }
+
+    @Test
+    void pathModifyingTest() throws InvocationTargetException,
```

Comment:
from my experience, in eo repo they usually just throw an Exception or an IOException and don't explicitly specify exception types (I might be wrong though, but I think it will better correspond with the overall codestyle)